### DISCUSSION TITLE
Starting Discussions! 📰

### DISCUSSION THREAD

**Discussion by LyubomirT** [LyubomirT/c.ai-addons#8] [Announcements]
Hi everyone! 👋

I'm excited to announce that we're setting up GitHub Discussions for our repository! This will give us a new way to discuss issues, share ideas, and collaborate on projects.

GitHub Discussions is a forum-style platform that's integrated with GitHub. This means that you can easily create and participate in discussions from within your GitHub workflow.

To get started, simply visit the Discussions tab on the repository page. You can then create a new discussion by clicking the "New discussion" button.

We encourage you to use GitHub Discussions to discuss anything related to the repository. This includes issues, feature requests, general questions, and more.

We hope that you'll find GitHub Discussions to be a valuable tool for our community.

Thanks,
Lyubomir.

**Comment by PencilDrawer12** [comment]
Hello!

In order to use the memories, the user, after manually generating memories, needs to just insert the memory strings in the chat every 4 or 5 comments before their next reply, right? Is there anything to be done when the AI sometimes talks about the things written in the memory strings instead of answering the reply of the user?

**Reply by LyubomirT** [reply]
Unfortunately, that's most likely the way the AI reacts to memory strings. This hugely depends on the specific character you're talking to. In the next update I'm planning to introduce custom memory string formats, maybe that could help.

**Reply by PencilDrawer12** [reply]
That is alright, I was just wondering if there was something wrong I was doing with the add on. I'll just skip the messages that reply to the memory string and see if it helps "train" the AI into not replying to them. Thank you for the quick reply ^^

**Reply by LyubomirT** [reply]
You're welcome @PencilDrawer12 :)

Should I ping you when the feature gets implemented so you can resolve the issue quickly?

**Reply by PencilDrawer12** [reply]
For sure! Thanks a lot ^^
